src/sass/messaging/partials/_message-compose.scss

```diff
@@ -272,8 +253,15 @@ button.msg-btn-save {
   }
 }
 
+.msg-compose-write {
+  @media (min-width: $medium-screen) {
+    border-left-color: green;
+    padding-left: 1.5rem;
+  }
+}
+
 .msg-write {
-  padding: 1.5rem 1.5rem 2rem 1.5rem;
+  padding: 1.5rem 1rem 2rem;
```

Comment:
With this, the message field doesn't line up with the other fields in mobile. It's slightly left (.5rem) of them. If this should have 1rem padding on the sides, the other fields (currently at 1.5rem for padding on sides) should as well.